app/register/form.tsx

```diff
@@ -16,9 +16,9 @@ export default function Form() {
             })
         })
 
-        if(!response?.ok) {
-            router.push('/');
-            router.refresh();
+        if(response?.ok) {
+            router.push('/login');
+            //router.refresh();
```

Comment:
I'm okay with your solution for now. We should probably modify the registration route to also create a session for the user. In which case, we would want to keep router.refresh and route the user to the /' page. But that's a separate issue!